specs/dwritecore/DWriteCore.md

```diff
@@ -170,8 +574,466 @@ DWRITE_BITMAP_DATA_BGRA32 TextRenderer::GetBitmapData(_In_ IDWriteBitmapRenderTa
 }
 ```
 
+## Using Font Selection APIs
+
+This section shows a complete console application that demonstrates the `IDWriteFontSet4::GetMatchingFonts`
+and `IDWriteFontSet4::GetDerivedFontAxisValues` methods. First let's include some headers:
+
+```c++
+#include <dwrite_core.h>
+#include <wil/com.h>
+#include <iostream>
+#include <string>
+#include <vector>
+```
+
+The `IDWriteFontSet4::GetMatchingFonts` method returns a list of fonts in priority order that match the
+specified family name and axis values. The following `MatchAxisValues` function outputs the parameters 
+to `GetMatchingFonts` and the list of matching fonts in the returned font set.
+
+```c++
+// Forward declarations of overloaded output operators used by MatchAxisValues.
+std::wostream& operator<<(std::wostream& out, DWRITE_FONT_AXIS_VALUE axisValue);
+std::wostream& operator<<(std::wostream& out, IDWriteFontFile& fileReference);
+std::wostream& operator<<(std::wostream& out, IDWriteFontFaceReference1& faceReference);
```

Comment:
DISCUSSION: These are Nano-COM references, where other code would have said IFoo*, this uses IFoo&

Reply:
I'll change these to take pointers instead of references. I agree a reference to a nano-COM interface is weird, as is passing a non-const reference to an operator.